### ISSUE TITLE
[Fully Automated Voiceovers] "Mark as Stale" option missing

### ISSUE THREAD

**Issue by Prapti-Kille** [oppia/product-operations-team#55]
While updating content in a state card, the voiceover update modal does not display the “Mark as Stale” option. This limits the ability to explicitly flag existing voiceovers as outdated when content changes.

Steps to Reproduce:

- Edit content in a state card (e.g., modify question text).
- Click on "Save Changes",  triggering the voiceover update modal.
- Observe the available options in the modal.

Observed Behavior:

<img width="285" height="182" alt="Image" src="https://github.com/user-attachments/assets/d6ba848e-e2c8-4644-b8d5-63b8c53ba61a" />

Expected Behavior:
The modal should include a “Mark as Stale” option to allow users to explicitly flag the existing voiceover as outdated due to content changes.

**Comment by NITISH084** [comment]
Hi @Nik-09 @Prapti-Kille Can you also tell which feature flag was on during this issue's testing?

<img width="2327" height="993" alt="Image" src="https://github.com/user-attachments/assets/e43b1857-b43a-4b22-b5c2-70c698dbd36c" />

From the code, it seems modifyTranslationsFeatureFlagIsEnabled, was enabled during this testing.  If that is the case then I believe still the current code changes won't show Mark as stale button, when modifyTranslationsFeatureFlagIsEnabled is enabled.

**Comment by Nik-09** [comment]
Hi @NITISH084, Yes, I confirm that  `exploration_editor_can_modify_translations` is enabled on the backup server. 

I've also updated the CUJ from row 11 to 24 in the LC tab [here](https://docs.google.com/spreadsheets/d/1XER4thLOWYoZxbgl6UNf1TOBb3elKMaNfIGkEAckr3w/edit?gid=1107995923#gid=1107995923) so that it correctly expects the behaviour for both the cases, i.e., when the feature flag is on and off.

Please let me know if you need any additional information.
Thanks!

**Comment by Prapti-Kille** [comment]
<img width="1440" height="900" alt="Image" src="https://github.com/user-attachments/assets/d2784864-bf44-4722-b3bd-53f69e435bfd" /> 

@Nik-09 Has the "Mark as Stale" option been deliberately taken out of the description and as an option?

**Comment by Prapti-Kille** [comment]
From my last conversation with @Nik-09, it seems like the functionality to whether to show "Mark as stale" or "Modify existing translation" is dependent on the feature flag exploration_editor_can_modify_translations. This feature flag is not related to voiceover regeneration functionality and was already enabled in the backup server. 

To test the "Mark as stale" button, we need to turn off the flag.

**Comment by Prapti-Kille** [comment]
<img width="799" height="464" alt="Image" src="https://github.com/user-attachments/assets/7eb0415d-ede7-4a23-af9d-271c7b066520" />

Similar minor formatting issues to issue #56, "Mark as Stale" in the bullet points needs to change to "Mark as stale", there needs to be a space between the ":" and "The" in the Note (Note:The -> Note: The), and the "Remove Existing Translations/Voiceovers" needs to be "Remove existing translations/voiceovers."

**Comment by seanlip** [comment]
@Prapti-Kille Should this be filed as a bug on the Tech Github board, since the next action should be by the dev team? https://github.com/oppia/oppia/issues
